unit_tests/Particle/ParticleBC.cpp

```diff
@@ -54,13 +54,10 @@ class ParticleBCTest<Parameters<T, ExecSpace, Rank<Dim>>> : public ::testing::Te
     }
 
     void checkResult(const ippl::Vector<T, Dim>& expected) {
-        T tol = (std::is_same_v<T, double>) ? 1e-15 : 1e-7;
-
         Kokkos::deep_copy(mirror, bunch->R.getView());
-
         for (int i = 0; i < nParticles; ++i) {
             for (size_t j = 0; j < Dim; ++j) {
-                ASSERT_NEAR(expected[j], mirror(i)[j], tol);
+                ASSERT_NEAR(expected[j], mirror(i)[j], tolerance<T> / 10);
```

Comment:
Why is the tolerance divided by 10? Shouldn't we then just provide another tolerance?

Reply:
I asked about this in the group chat. We didn't reach a conclusion on a specific reason why this test had a lower tolerance than all the others. In the PR I just left it like this so that the expression reflects the nature of the test, but we can also make a new constant and call it `lower_tolerance` or similar.

Reply:
Ok. I see.